x-pack/legacy/plugins/ml/public/application/datavisualizer/index_based/components/fields_panel/fields_panel.tsx

```diff
@@ -100,7 +100,7 @@ export const FieldsPanel: FC<Props> = ({
                   />
                 }
               >
-                <EuiBadge>
+                <EuiBadge title="">
                   <b>{fieldVisConfigs.length}</b>
                 </EuiBadge>
```

Comment:
Is there a reason we added the `title` prop in?
And can we remove the `<b>` tag.  Let's stick to the default weight in the `EuiBadge`.

Reply:
the badge is wrapped with a tooltip, so we decided there is no need in the title (by default it's a content of the badge)